tests/manage/pv_services/pv_encryption/test_pvc_with_tc_overrie_vn.py

```diff
@@ -0,0 +1,178 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+    bugzilla,
+)
+
+from ocs_ci.helpers.helpers import (
+    create_unique_resource_name,
+    create_pods
+)
+
+from ocs_ci.ocs.exceptions import ResourceNotFoundError
+from ocs_ci.utility import kms
+
+log = logging.getLogger(__name__)
+
+kmsprovider = constants.VAULT_KMS_PROVIDER
+argnames = ["kv_version", "kms_provider", "use_vault_namespace"]
+if config.ENV_DATA.get("vault_hcp"):
+    argvalues = [
+        pytest.param(
+            "v1", kmsprovider, True, marks=pytest.mark.polarion_id("OCS-4639")
+        ),
+        pytest.param(
+            "v2", kmsprovider, True, marks=pytest.mark.polarion_id("OCS-4641")
+        ),
+    ]
+else:
+    argvalues = [
+        pytest.param(
+            "v1", kmsprovider, False, marks=pytest.mark.polarion_id("OCS-4638")
+        ),
+        pytest.param(
+            "v2", kmsprovider, False, marks=pytest.mark.polarion_id("OCS-4640")
+        ),
+    ]
+
+
+@pytest.mark.parametrize(
+    argnames=argnames,
+    argvalues=argvalues,
+)
+@tier1
+@skipif_ocs_version("<4.8")
```

Comment:
Any reason for skipped this test for < 4.8? I see the BZ was reported and fixed in 4.10